### PR TITLE
Add script for installing the interop server locally

### PR DESCRIPTION
**Pull request by foochuanwei**
The script can only be used to install. It is not able to update the server.

You may have comments to make about the location of the script and how it is supposed to be run.

### CODE REVIEW

**Comment by anassinator** [comment]
I think this should be in the `scripts` directory as an executable such that people can 

```bash
rosrun interop install-server
```

I also think there should be corresponding scripts to uninstall, start/stop the server.

Finally, I'm not sure if we want this to be in this package.

File: tools/install-auvsi-interop [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+# There are few concrete reasons for enforcing this, but check anyway.
+# Root has a UID of 0.
+if [ $EUID == 0 ]; then
+    echo "Do not run this script as root (don't use sudo)."
+    exit 1
+fi
+
+echo 'Ensuring that git is installed and up to date.'
+sudo apt-get update
+sudo apt-get install -y git
```
Comment (anassinator) [inline tools/install-auvsi-interop]: this should only be done if `git` is not already installed

File: tools/install-auvsi-interop [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+# https://auvsi-suas-competition-interoperability-system.readthedocs.io/en/latest/getting_started.html
+
+set -e  # Exit on the first error.
+
+# This script should not be run as root.
+# There are few concrete reasons for enforcing this, but check anyway.
+# Root has a UID of 0.
+if [ $EUID == 0 ]; then
+    echo "Do not run this script as root (don't use sudo)."
+    exit 1
```
Comment (anassinator) [inline tools/install-auvsi-interop]: 2 spaces

File: tools/install-auvsi-interop [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+# the one provided by McGill Robotics at interop.mcgillrobotics.com.
+# This script is based on:
+# https://auvsi-suas-competition-interoperability-system.readthedocs.io/en/latest/getting_started.html
+
+set -e  # Exit on the first error.
+
+# This script should not be run as root.
+# There are few concrete reasons for enforcing this, but check anyway.
+# Root has a UID of 0.
+if [ $EUID == 0 ]; then
```
Comment (anassinator) [inline tools/install-auvsi-interop]: ${EUID}

File: tools/install-auvsi-interop [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    echo "Do not run this script as root (don't use sudo)."
+    exit 1
+fi
+
+echo 'Ensuring that git is installed and up to date.'
```
Comment (anassinator) [inline tools/install-auvsi-interop]: be consistent with `"` vs `'`

File: tools/install-auvsi-interop [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+echo
+
+echo 'Cloning the AUVSI interoperability server git repository.'
+# Need sudo because some files may have been created by root.
+sudo rm -rf /tmp/auvsi-interop/
+git clone https://github.com/auvsi-suas/interop.git /tmp/auvsi-interop/
+echo
+
+echo 'Setting up the computer to run the server.'
+cd /tmp/auvsi-interop/
```
Comment (anassinator) [inline tools/install-auvsi-interop]: maybe the location should be an argument

**Comment by foochuanwei** [comment]
Agree with all your points.
I was looking to get these scripts version controlled because they are useful on interop.mcgillrobotics.com, and can be useful for future flight tests.

**Pull request closed without merging**